### PR TITLE
added notifications for buyer video consultation request, banker resp…

### PR DESCRIPTION
**Pull request by ChaitanyaV98**
…onses, buyer payments

### CODE REVIEW

File: app/loco/resources/VideoConsultation.json
Comment (rajeshn95) [inline app/loco/resources/VideoConsultation.json:1]: @ChaitanyaV98 Why VideoConsultation resource is created again?

**Review by rajeshn95 — CHANGES_REQUESTED** [review]

File: app/loco/resources/VideoConsultation.json
Comment (ChaitanyaV98) [reply app/loco/resources/VideoConsultation.json:1]: file name is named in Video-Consultation format @rajeshn95, so when using the resource name in hooks file it is showing error @rajeshn95.

Comment (ChaitanyaV98) [reply app/loco/resources/VideoConsultation.json:1]: so thats why we have created a new file and copied the code here, shall I remove old file?

Comment (rajeshn95) [reply app/loco/resources/VideoConsultation.json:1]: What was the error?

Comment (ChaitanyaV98) [reply app/loco/resources/VideoConsultation.json:1]: function naming will not allow "-" @rajeshn95

Comment (rajeshn95) [reply app/loco/resources/VideoConsultation.json:1]: This is a resource file, where exactly is this error coming?

**Pull request merged**